tobac/utils/general.py

```diff
@@ -168,12 +172,14 @@ def add_coordinates_3D(
         Typically, 'longitude','latitude','x_projection_coordinate','y_projection_coordinate',
         and 'altitude' (if 3D) are the coordinates that we expect, although this function
         will happily interpolate along any dimension coordinates you give.
-    vertical_coord: str or int
-        Name or axis number of the vertical coordinate. If 'auto', tries to auto-detect.
+    vertical_coord: str or None
```

Comment:
update documentation here to be consistent with the latest changes in `RC_v1.5.0`